src/screens/LpnDetail/Index.tsx

```diff
@@ -12,6 +12,15 @@ import Button from "../../components/Button";
 import PrintModal from "../../components/PrintModal";
 import showPopup from "../../components/Popup";
 import {getProductByIdAction} from "../../redux/actions/products";
+import SelectDropdown from "react-native-select-dropdown";
+
+
+const containerStatus = ["Open", "Packing", "Packed", "Loaded"]
```

Comment:
Statuses from the ticket were: ` (“PACKING”, “PACKED”, “LOADED”, “UNLOADED”, “UNPACKING”, “UNPACKED”)`. Additionally, I think these should be pulled from the api.